psalm-strict.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <psalm
-    totallyTyped="true"
+    errorLevel="1"
```

Comment:
The `totallyTyped` setting is deprecated, see https://psalm.dev/docs/running_psalm/configuration/#totallytyped